test/container_test.py

```diff
@@ -111,7 +217,7 @@ def test_set_volume_unit_conv(self):
         c.well(1).set_volume(".1:milliliter")
         self.assertTrue(c.well(1).volume == Unit(100, "microliter"))
         with self.assertRaises(ValueError):
-            c.well(2).set_volume("1:liter")
+            c.well(2).set_volume("1:milliliter")
```

Comment:
Will do. I think it's useful to test this, though; the ml test correctly tests the maximum-volume check in set_volume, since the dummy_type well capacity is 200 µl, so you'll lose coverage if we remove this test.